### PR TITLE
Add Quantum Kittens Notepad custom problem set

### PR DESCRIPTION
**Pull request by quantum-kittens**
## Changes

Add Quantum Kittens Notepad custom problem set.

This set acts as a coding scratchpad for readers of [Quantum Kittens](quantumkittens.online) to learn and play with basic Qiskit code, sans graders.

Each heading should be a new page.

### CODE REVIEW

File: notebooks/problem-sets/quantum_kittens_notepad.ipynb [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "goals": [
+     {
+      "id": "qiskit-beginner-1"
+     }
+    ],
+    "grader_answer": "circuit",
```
Comment (vabarbosa) [inline notebooks/problem-sets/quantum_kittens_notepad.ipynb:87]: this should be `marble_circuit` since that is the variable name used in the code cell

**Review by vabarbosa — CHANGES_REQUESTED** [review]
@quantum-kittens this looks good. there is just one issue (see comment below). thanks!

File: notebooks/problem-sets/quantum_kittens_notepad.ipynb (hunk 1 above)
Comment (quantum-kittens) [reply notebooks/problem-sets/quantum_kittens_notepad.ipynb]: @vabarbosa Thanks for spotting this. There's no grader, so I've updated the metadata as follows:

- Changed `grader_id` and `grader_answer` to blank
- Changed id to qk-notepad

Please let me know if this is okay.

File: notebooks/problem-sets/quantum_kittens_notepad.ipynb [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "goals": [
+     {
+      "id": "qk-notepad"
+     }
+    ],
+    "grader_answer": "",
+    "grader_id": ""
```
Comment (vabarbosa) [inline notebooks/problem-sets/quantum_kittens_notepad.ipynb:88]:
```suggestion
    ]
```

**Review by vabarbosa — APPROVED** [review]
thank you!

**Pull request merged**